lib/toadScheduler.ts

```diff
@@ -54,6 +66,16 @@ export class ToadScheduler {
     this.engines.cronJobEngine.add(job)
   }
 
+  addJob(job: Job): void {
+    if (job instanceof SimpleIntervalJob || job instanceof LongIntervalJob) {
```

Comment:
instanceof check is notoriously unrealiable and will not work correctly when comparing instances created in a different realm (e. g. if you are using a job created inside a shared library in an app that depends both on a shared library and on toad-scheduler). I would recommend introducing some id field on the job, and using it for a proper type guard